hack/cluster-openshift.sh

```diff
@@ -177,13 +187,21 @@ OPENSHIFT_BIN_PATH="${OPENSHIFT_BIN_PATH:=${HOME}/bin}"
 
 # This is the IP address where OpenShift will bind its master.
 # This should be a valid IP address for the machine where OpenShift is installed.
-# NOTE: Do not use any IP address within the loopback range of 127.0.0.x.
-OPENSHIFT_IP_ADDRESS=${OPENSHIFT_IP_ADDRESS:-`echo $(ip -f inet addr | grep 'state UP' -A1 | tail -n1 | awk '{print $2}' | cut -f1 -d'/')`}
+
+if [ ! "$OPENSHIFT_IP_ADDRESS" ] ; then
+  OPENSHIFT_IP_ADDRESS=${DEFAULT_OPENSHIFT_IP_ADDRESS}
+fi
 
 # The version is the tag from the openshift-istio/origin release builds.
 # The platform is either "linux" or "darwin".
 OS_ISTIO_OC_DOWNLOAD_VERSION="${OS_ISTIO_OC_DOWNLOAD_VERSION:-istio-3.9-0.8.0-alpha4}"
-OS_ISTIO_OC_DOWNLOAD_PLATFORM="${OS_ISTIO_OC_DOWNLOAD_PLATFORM:-linux}"
+DEFAULT_OS_VERSION=linux
+DETECTED_OS_VERSION=`uname | tr '[:upper:]' '[:lower:]'`
+if [ "${DETECTED_OS_VERSION}" = "linux" -o "${DETECTED_OS_VERSION}" = "darwin" ] ; then
+  DEFAULT_OS_VERSION=${DETECTED_OS_VERSION}
+  debug "he operating system has been detected as ${DEFAULT_OS_VERSION}"
```

Comment:
Missing the `T` for `The`

Reply:
@josejulio Thanks, I really hate that kind of silly error.